website/docs/r/database_user.html.markdown

```diff
@@ -76,10 +76,10 @@ In addition to all arguments above, the following attributes are exported:
 
```

Comment:
Missing some needed changes to database_name, same as my comments in the data source docs and also would be great to have both a SCRAM authentication and X509 example.   Also doesn't this need the X509_type or did you set that w/ the X509 resource (I think it is needed here?) ?

Reply:
Yes, It would be great, but I want first to merge the x509 resource to be able to create a test with both resources and later put the documentation examples.

Reply:
That works for me.  Just make sure any changes get updated doc.